doc/design/diagrams/request_response/request_response_header.puml

```diff
@@ -15,9 +15,6 @@ class RpcBaseHeader {
 class RequestHeader {
   + RequestHeader(uniqueClientQueueId: cxx::UniqueId, lastKnownQueueIndex: uint32_t)
   + setSequenceId(sequenceId: int64_t): void
-  + setFireAndForget(): void
```

Comment:
I prefer `+ - #` for private, public, protected. You can use `skinparam classAttributeIconSize 0` to force it.

Reply:
Not sure about this. I tried it and found it distracting since it is rendered with the text color and not colorful like the diff above. Let's the second reviewer decide

Reply:
Not sure if  I got it. But everything is correct and how it is normally plotted with puml. You want to have `+ - #` instead of the default symbols  @mossmaurice? For me it is fine as default puml

Reply:
@budrus Exactly, always have to look up what the symbols mean :smile: